### PR TITLE
Try seek blob stream position to begin

### PR DESCRIPTION
**Pull request by JadynWong**
Use the file system or azure blob the blob provider, the Position of the returned stream is at the end.

### CODE REVIEW

**Comment by maliming** [comment]
We should change `blob provider` instead of  `CmsKit`.

**Review by cotur — CHANGES_REQUESTED** [review]
Hi @JadynWong,

as @maliming said, we need to upgrade the `Blob Providers` instead of updating it at the `Cms-Kit` module.

File: framework/src/Volo.Abp.BlobStoring/Volo/Abp/BlobStoring/BlobContainer.cs [hunk 1]
```diff
@@ -214,5 +214,22 @@ public class BlobContainer : IBlobContainer
...
+            }
+
+            if (!stream.CanSeek)
+            {
+                return stream;
+            }
+
+            stream.Seek(0, SeekOrigin.Begin);
+
+            return stream;
```
Comment (rqx110) [inline framework/src/Volo.Abp.BlobStoring/Volo/Abp/BlobStoring/BlobContainer.cs]: can simplified：
```suggestion            
            if (stream != null && stream.CanSeek)
            {
                stream.Seek(0, SeekOrigin.Begin);
            }

            return stream;
```

**Comment by maliming** [comment]
We need to change the `Stream`  for all `BlobProviders`.

https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.BlobStoring.FileSystem/Volo/Abp/BlobStoring/FileSystem/FileSystemBlobProvider.cs#L79
https://github.com/abpframework/abp/blob/dev/framework/src/Volo.Abp.BlobStoring.Azure/Volo/Abp/BlobStoring/Azure/AzureBlobProvider.cs#L72
https://github.com/abpframework/abp/blob/dev/modules/blob-storing-database/src/Volo.Abp.BlobStoring.Database.Domain/Volo/Abp/BlobStoring/Database/DatabaseBlobProvider.cs#L122

**Comment by JadynWong** [comment]
Hi @maliming 

I have some questions.
Now I try to handle it in `BlobContainer`'s `GetOrNullAsync`. Users usually use the `IBlobContainer` to manipulate the blob, and the `BlobProvider` only as an implementation.
Do we need to move the `TrySeekStreamToBegin` method to the `BlobProviderBase`, and handle it in each Provider?

**Comment by maliming** [comment]
We are using `MemoryStream`, we can change it directly. 

No need for the `TrySeekStreamToBegin` method

**Comment by JadynWong** [comment]
`DatabaseBlobProvider` using `byte[]` does not require it, it is default at start position

**Review by maliming — APPROVED** [review]

**Review by cotur — APPROVED** [review]

**Pull request merged**